client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -171,13 +172,16 @@ class AuthorMappingPane extends React.PureComponent {
 	}
 }
 
-const withTotalUsers = ( Component ) => ( props ) => {
-	const { siteId } = props;
-	const { data } = useUsersQuery( siteId, {}, { refetchOnWindowFocus: false } );
+const withTotalUsers = createHigherOrderComponent(
+	( Component ) => ( props ) => {
+		const { siteId } = props;
+		const { data } = useUsersQuery( siteId );
 
-	const totalUsers = data?.total ?? 0;
+		const totalUsers = data?.total ?? 0;
 
-	return <Component totalUsers={ totalUsers } { ...props } />;
-};
+		return <Component totalUsers={ totalUsers } { ...props } />;
+	},
+	'withTotalUsers'
```

Comment:
Nit: it's much more common to have the component name with a capital first letter than with a small one.

Reply:
It seems fairly equal from what I see. I don't have a strong opinion on that and used both in the past. I'd merge it as is but I'm happy to follow up in case we decide to go with either for all of the usages.